src/schemas/capabilities.json

```diff
@@ -0,0 +1,88 @@
+{
+    "$id": "https://meta.comcast.com/firebolt/capabilities",
+    "title": "Capabilities",
+    "oneOf": [
+        {
+            "$ref": "#/definitions/Capability"
+        },
+        {
+            "$ref": "#/definitions/Role"
+        },
+        {
+            "$ref": "#/definitions/DenyReason"
+        },
+        {
+            "$ref": "#/definitions/CapabilityInfo"
+        }
+    ],
+    "definitions": {
+        "Capability": {
+            "type": "string",
+            "description": "A Capability is a discrete unit of functionality that a Firebolt device might be able to perform.",
+            "pattern": "xrn:firebolt:capability:([a-z0-9]+)((:[a-z0-9]+)?)"
+        },
+        "Role": {
+            "title": "Role",
+            "description": "Role provides access level for the app for a given capability.",
+            "type": "string",
+            "enum": ["use", "manage", "provide"]
+        },
+        "DenyReason": {
+            "title": "DenyReason",
+            "description": "Reasons why a Capability might not be invokabl",
+            "type": "string",
+            "enum": [
+                "unpermitted",
+                "unsupported",
+                "disabled",
```

Comment:
I think it makes sense to include "disabled" since it will be a very common reason, but just keep in mind that it's a sub-set of "unavailable".

Maybe we should move it to an open-ended string for additional details, like "message" or "sub-code"